File: CS/PersonalOrganizer/ContactContextDataModel/Model/IContactRepository.cs
using System;
using System.Linq;
using System.Data;
using System.Data.Entity;
using System.Linq.Expressions;
using System.Collections.Generic;
using PersonalOrganizer.Common.Utils;
using PersonalOrganizer.Model;
using PersonalOrganizer.Common.DataModel;
using PersonalOrganizer.Common.DataModel.EntityFramework;

namespace PersonalOrganizer.ContactContextDataModel {
    public interface IContactRepository : IRepository<Contact, int> {
    }
}

File: VB/PersonalOrganizer/ContactContextDataModel/Model/IContactRepository.vb
Imports System
Imports System.Linq
Imports System.Data
Imports System.Data.Entity
Imports System.Linq.Expressions
Imports System.Collections.Generic
Imports Common.Utils
Imports PersonalOrganizer.Model
Imports Common.DataModel
Imports Common.DataModel.EntityFramework
Namespace ContactContextDataModel
    Public Interface IContactRepository
        Inherits IRepository(Of Contact, Integer)
    End Interface
End Namespace
